Automattic/fresh-data
# Update deps for packages and examples

Note: Even though the line count is really high, it's all in package-lock.json files.
There are no "real" code changes in this PR.

This updates the dependencies for the packages and examples.
It also sets the examples to depend on a specific version number instead
of whatever is in the checked-out repository, which helps decouple the
examples a little from the packages. This allows them to be updated in
subsequent PRs as needed after updating the packages themselves.

To Test:
1. `npm install`
2. `npm run bootstrap`
3. `npm run build`
4. In each examples subdirectory: `npm install`, then `npm start` to check each example app.
